src/ucs/config/parser.c

```diff
@@ -865,6 +906,14 @@ ucs_config_parser_set_default_values(void *opts, ucs_config_field_t *fields)
             continue;
         }
 
+        if (ucs_config_is_deprecated_field(field)) {
```

Comment:
user has to set UCS_CONFIG_TYPE_DEPRECATED in the table

Reply:
done